fix(python,docs): add examples on how to use python sdk (#498)
matter-labs/zksync-web-era-docs: docs/api/python/accounts.md

# Accounts: overview

> The zkSync Python SDK, has a few methods to generate signature and verify messages.

Reviewer: ```suggestion
The zkSync Python SDK, has a few methods that generate signature and verify messages.
```

# Accounts: overview

> The zkSync Python SDK has a few methods that generate signature and verify messages.

Reviewer: ```suggestion
The zkSync Python SDK has a method that generates a signature and a few methods that verify messages.
```

Reviewer: Yes, be specific about this otherwise they are both singular, or both plural. And if you use the plural as before, it's OK, it doesn't matter if it's just one.

Author: `The zkSync Python SDK has a method that generates a signature and a method that verify messages.` is correct.

Reviewer: Thank you, amended :)

> The zkSync Python SDK, has a few methods that generate signature and verify messages.

Reviewer: ```suggestion
The zkSync Python SDK has a few methods that generate signature and verify messages.
```
